pkgs/by-name/se/servarr-ffmpeg/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  ffmpeg-headless,
+  fetchFromGitHub,
+  fetchpatch2,
+  lib,
+}:
+
+let
+  version = "5.1.4";
+in
+
+(ffmpeg-headless.override {
+  inherit version; # Important! This sets the ABI.
+
+  # Fetch commit hash from this repository: https://github.com/Servarr/ffmpeg-build
+  # Compare build logs to upstream logs here: https://dev.azure.com/Servarr/Servarr/_build?definitionId=15
+  source = fetchFromGitHub {
+    owner = "Servarr";
+    repo = "FFmpeg";
+    rev = "e9230b4c9027435dd402a68833f144643a3df43a";
+    hash = "sha256-oMIblMOnnYpKvYeleCZpFZURGVc3fDAlYpOJu+u7HkU=";
+  };
+
+  buildFfmpeg = false;
+  buildFfplay = false;
+  buildAvdevice = false;
+  buildAvfilter = false;
+  buildPostproc = false;
+  buildSwresample = false;
+  buildSwscale = false;
+
+  withAlsa = false;
+  withAmf = false;
+  withAom = false;
+  withAss = false;
+  withBluray = false;
+  withBzlib = false;
+  withCudaLLVM = false;
+  withCuvid = false;
+  withDrm = false;
+  withNvcodec = false;
+  withFontconfig = false;
+  withFreetype = false;
+  withFribidi = false;
+  withGnutls = false;
+  withIconv = false;
+  withLzma = false;
+  withMp3lame = false;
+  withOpencl = false;
+  withOpenjpeg = false;
+  withOpenmpt = false;
+  withOpus = false;
+  withRist = false;
+  withSoxr = false;
+  withSpeex = false;
+  withSrt = false;
+  withSsh = false;
+  withSvtav1 = false;
+  withTheora = false;
+  withV4l2 = false;
+  withVaapi = false;
+  withVidStab = false;
+  withVorbis = false;
+  withVpx = false;
+  withVulkan = false;
+  withWebp = false;
+  withX264 = false;
+  withX265 = false;
+  withXml2 = false;
+  withXvid = false;
+  withZimg = false;
+  withZlib = false;
+  withZvbi = false;
+}).overrideAttrs
```

Comment:
I have a feeling @emilazy might be opposed to having a package definition only consist of an override and overrideAttrs, but I see that you copied this over from [jellyfin-ffmpeg](https://github.com/NixOS/nixpkgs/blob/nixos-25.05/pkgs/by-name/je/jellyfin-ffmpeg/package.nix) is implemented. Maybe one of the ffmpeg package maintainers (also @Atemu and @jopejoe1, besides Emily) can chime in about their stance on in-tree ffmpeg forks and what they are supposed to look like?

Reply:
I'll assume that this is fine in its current state, as it closely resembles the jellyfin-ffmpeg package.